app/app.go

```diff
@@ -140,6 +140,8 @@ const (
 	Name = "paloma"
 
 	wasmAvailableCapabilities = "iterator,staking,stargate,paloma"
+
+	minimumPigeonVersion = "v1.4.0"
```

Comment:
If this is set here, how is it changed at governance for the upgrade to say pigeon `v.1.5.0`

Reply:
When we have a new version of Paloma, there are two possibilities:
1. The new version is still compatible with all Pigeon versions down to v1.4.0
    * No Pigeon update required, no change to this variable.  Pigeons can use any version equal to or greater than this number
2. The new version requires a Pigeon update
    * We update this variable to indicate the lowest pigeon version compatible with the new Paloma version.  This code goes through Paloma's upgrade proposal.  Pigeons can upgrade early if they want

Reply:
Okay. so Anyone can fork paloma, update app.go, and submit their version for the new minimum pigeon. Yes?

Reply:
I guess this is easier than having paloma VOTE on the minimum pigeon required, but it does remove governance for selecting the version.

Reply:
Volume changing the number and then forcing a vote is kinda really volum controlling all the software. I think we need the version to be SET in governance from a community vote.

Reply:
This minimum version does go through Paloma's voting though.  It's specified in the code, and the code is voted on during the upgrade proposal.  If the community doesn't like the change, they can vote against it just like any other code change in Paloma.